test/parallel/test-event-emitter-remove-listeners.js

```diff
@@ -102,3 +102,15 @@ e6.emit('hello');
 
 // Interal listener array [listener3]
 e6.emit('hello');
+
+const e7 = new events.EventEmitter();
+
+const listener5 = () => {};
+const listener6 = common.mustCall((eventName, listener) => {
```

Comment:
might be clearer to just inline `listener6` with the `removeListener` line below instead of creating a separate decl for it